config/samples/lightgbm-dist/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:16.04
+
+ARG CONDA_DIR=/opt/conda
+ENV PATH $CONDA_DIR/bin:$PATH
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    ca-certificates \
+    cmake \
+    build-essential \
+    gcc \
+    g++ \
+    git \
+    wget && \
+    # python environment
+    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
+    /bin/bash Miniconda3-latest-Linux-x86_64.sh -f -b -p $CONDA_DIR && \
+    export PATH="$CONDA_DIR/bin:$PATH" && \
+    conda config --set always_yes yes --set changeps1 no && \
+    # lightgbm
+    conda install -q -y numpy scipy scikit-learn pandas && \
+    git clone --recursive --branch stable --depth 1 https://github.com/Microsoft/LightGBM && \
```

Comment:
Would it be necessary to check out a particular version in case any of the build steps have changed in the future?

Reply:
[Versions ok](https://github.com/kubeflow/xgboost-operator/pull/103/commits/947773e1598ba184193cd0fac592f04ff294940b)